config/default.go

```diff
@@ -54,6 +54,7 @@ PurgeByExtDelInterval = "1m"
 MustForgeAtSlotDeadline = true
 IgnoreSlotCommitment = true
 ForgeOncePerSlotIfTxs = false
+ProverWaitReadTimeout = "20s"
```

Comment:
Please add the new parameter to the documentation:
https://github.com/hermeznetwork/hermez-node/blob/develop/config/README.md

Reply:
done!